open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/EngineHostServicesConfig.java

```diff
@@ -23,17 +24,21 @@
  *         server running the Governance Engine OMAS that provides configuration for the governance engines
  *         as well as the governance actions that drive the automated execution of governance requests.</li>
  *
- *     <li>An array of EngineServiceConfig properties, one for each engine service to run.</li>
+ *     <li>An array of Engine properties, one for each engine to run.</li>
+ *
+ *     <li>An optional array of EngineServiceConfig properties, one for each engine service to run.</li>
  * </ul>
  */
 @JsonAutoDetect(getterVisibility=PUBLIC_ONLY, setterVisibility=PUBLIC_ONLY, fieldVisibility=NONE)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown=true)
 public class EngineHostServicesConfig extends OMAGServerClientConfig
 {
-    private static final long    serialVersionUID = 1L;
+    @Serial
```

Comment:
I wonder why serial annotation is only on this field

Reply:
As I work my way through the beans removing Serializable, these annotations will be removed.